README.md

```diff
@@ -1,23 +1,25 @@
 # buoy
+Accessible Vue.js UI component library
 
-![Buoy logo](./buoylogo.png)
+[![CircleCI](https://circleci.com/gh/fanappics/buoy/tree/master.svg?style=svg)](https://circleci.com/gh/fanappics/buoy/tree/master)
 
-Accessible Vue.js UI library
+<img src="https://raw.githubusercontent.com/fanappics/buoy/master/buoylogo.png" alt="Buoy Logo" width="300" height="300">
 
-## Goals
+## Goal
 
-The goal of the Buoy project is to create a beautiful, functional UI library for Vue.js that makes it easy for developers to follow the [Web Content Accessibility Guidelines guidelines](https://www.w3.org/WAI/intro/wcag).
+The Buoy project will create a beautiful, functional UI library for Vue.js that will make it easy for developers to follow the [Web Content Accessibility Guidelines guidelines](https://www.w3.org/WAI/intro/wcag).
 
-https://pattern-library.dequelabs.com is a resource for an accessible UI library we can use for an example of accessibility techniques.
+## Principles
```

Comment:
inspirations?

stolen ideas?

things we like?

Reply:
I like `Inspirations`

Reply:
Though `pa11y` is more a tool that we use, but the others all fit

Reply:
Actually, do we have a space for tools in the readme?

Reply:
There's scaffolding commands at the bottom and, presumably, developers know to look in the `package.json` and `circle.yml`.

I can add those once we define what the "important commands" are

Reply:
Sure!